runtime/src/snapshot_utils.rs

```diff
@@ -80,7 +80,7 @@ pub const SNAPSHOT_FULL_SNAPSHOT_SLOT_FILENAME: &str = "full_snapshot_slot";
 pub const BANK_SNAPSHOTS_DIR: &str = "snapshots";
 pub const MAX_SNAPSHOT_DATA_FILE_SIZE: u64 = 32 * 1024 * 1024 * 1024; // 32 GiB
 const MAX_SNAPSHOT_VERSION_FILE_SIZE: u64 = 8; // byte
-const SNAPSHOT_FASTBOOT_VERSION: Version = Version::new(1, 0, 0);
+const SNAPSHOT_FASTBOOT_VERSION: Version = Version::new(2, 0, 0);
```

Comment:
Should the fastboot version be contingent on if marking obsolete accounts is enabled or not?

Reply:
Can we add a document for the version history: 
// Version 1.0.0: Initial fastboot support ...
// Version 2.0.0: Added obsolete accounts support, removed state_complete/storages_flushed files ...